ethereum/dbstack/db_stack.go

```diff
@@ -1,91 +1,168 @@
 package dbstack
 
 import (
+	"fmt"
 	"sync"
 
 	"github.com/0xProject/0x-mesh/ethereum/miniheader"
 	"github.com/0xProject/0x-mesh/meshdb"
 )
 
-// DBStack allows performing basic stack operations on a stack of meshdb.MiniHeaders stored
-// in the DB backing our meshdb package.
-type DBStack struct {
-	// TODO(albrow): Use Transactions when db supports them instead of a mutex
-	// here. There are cases where we need to make sure no modifications are made
-	// to the database in between a read/write or read/delete.
-	mut    sync.Mutex
-	meshDB *meshdb.MeshDB
-	limit  int
+type updateType int
```

Comment:
Can we de-dupe the definitions for `updateType` and `update`? Currently defined both here and in __ethereum/blockwatch/simple_stack.go__.